CHANGELOG_PENDING.md

```diff
@@ -35,3 +35,5 @@
 - \#2018 Only mark tickets for failed transactions as redeemed when there is an error checking the transaction (@yondonfu)
 
 #### Transcoder
+
```

Comment:
IIUC [as of this ffmpeg PR](https://github.com/livepeer/FFmpeg/pull/12/files#r705508457), go-livepeer no longer supports the Kepler series of Nvidia GPUs. I know we've already updated `doc/gpu.md` with a note that go-livepeer supports the Maxwell series and beyond, but since dropping Kepler is a breaking change I think we should make note of the breaking change in the changelog.

Let's add the following to the top of CHANGELOG_PENDING.md above Features and Bug Fixes:

```
### Breaking Changes 🚨🚨

- Insert note about the breaking change here.
```